### PR TITLE
eliminate duplicate code for CheckResult class

### LINKED ISSUE #1112: [Feature][common] eliminate duplicate code for CheckResult class
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/incubator-seatunnel/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

 Refactor CheckResult.java to eliminate duplicate code

### Usage Scenario

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by yx91490**
## Purpose of this pull request
close #1112 

## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason:
* [x] If any new Jar binary package adding in you PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/developement/NewLicenseGuide.md)
* [x] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

**Comment by RickyHuo** [comment]
PTAL @simon824

File: seatunnel-common/src/main/java/org/apache/seatunnel/common/config/CheckResult.java [hunk 1]
```diff
@@ -22,12 +22,36 @@
 @Data
 public class CheckResult {
 
+    private static final CheckResult SUCCESS = new CheckResult(true, "");
+
```
Comment (simon824) [inline seatunnel-common/src/main/java/org/apache/seatunnel/common/config/CheckResult.java:26]: we can add some success msg.

Comment (wuchunfu) [reply seatunnel-common/src/main/java/org/apache/seatunnel/common/config/CheckResult.java:26]: > we can add some success msg.

+1

Comment (yx91490) [reply seatunnel-common/src/main/java/org/apache/seatunnel/common/config/CheckResult.java:26]: no need, the success msg no references.

Comment (CalvinKirs) [reply seatunnel-common/src/main/java/org/apache/seatunnel/common/config/CheckResult.java:26]: yup

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Pull request merged**